src/setupTests.ts

```diff
@@ -0,0 +1,20 @@
+import Adapter from 'enzyme-adapter-react-16';
+import { configure } from 'enzyme';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { GlobalWithFetchMock } from 'jest-fetch-mock';
+// import './common/test/testi18nInit';
+
+const customGlobal: GlobalWithFetchMock = global as GlobalWithFetchMock;
+// eslint-disable-next-line import/no-extraneous-dependencies
+customGlobal.fetch = require('jest-fetch-mock');
+
+customGlobal.fetchMock = customGlobal.fetch;
+
+configure({ adapter: new Adapter() });
+
+jest.mock('react-router', () => ({
+  ...jest.requireActual('react-router'),
+  useHistory: (): Record<string, Function> => ({
+    push: jest.fn()
+  })
+}));
```

Comment:
What are test related files needed for, if there aren't any tests?

Reply:
Tests added